test/event_test.rb

```diff
@@ -0,0 +1,89 @@
+require 'pry'
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/event'
+
+class EventTest < Minitest::Test
+
+  def test_it_exists
+    event = Event.new("Gymnastics", [32, 22, 24, 26])
+    assert_instance_of Event, event
+  end
+
+  def test_max_age
+    event = Event.new("Gymnastics", [32, 22, 24, 26])
+    expected = [32, 22, 24, 26].max
```

Comment:
Remember, you want to be as explicit as possible with your expected values. Instead of this, do `expected = 32`